pkg/apis/flowcontrol/types.go

```diff
@@ -0,0 +1,278 @@
+/*
```

Comment:
Where are the external types? I would prefer to comment on those.

Reply:
+1

Reply:
This has been resolved.